.github/workflows/formatting.yml

```diff
@@ -31,17 +31,9 @@ jobs:
           pip install -U --upgrade pip
           pip install -U .[all,dev]
 
-      # Run the yapf format test
-      - name: Yapf
-        run: yapf --diff -r .
-
-      # Run the flake8 format test
-      - name: Flake8
-        run: flake8 .
-
-      # Run the isort format test
-      - name: Isort
-        run: isort --check-only .
+      # Run the ruff format test
+      - name: Ruff
+        run: ruff check .
```

Comment:
We use `ruff check src/` in the Python client

Reply:
It was just because there were not unit tests for the Python client when we introduced ruff. But I think it might be better to specify the directories explicitly so I made a change like that!!